app/src/main/java/com/example/footballmanager/network/Fixture.kt

```diff
@@ -1,14 +1,31 @@
 package com.example.footballmanager.network
 
 import com.google.gson.annotations.SerializedName
+import java.io.Serial
 
 data class FixtureDataWrapper(
     @SerializedName("response") var responseBody: ArrayList<ResponseBody> = arrayListOf()
 )
 
 data class ResponseBody(
     @SerializedName("teams") var teams: Teams? = Teams(),
-    @SerializedName("goals") var goals: Goals? = Goals()
+    @SerializedName("goals") var goals: Goals? = Goals(),
```

Comment:
`@SerializedName` annotations are redundant in this case as json parameters and property names match.
This annotation is useful eg. when json param is `match_result` and property name  is `matchResult` (btw. read about [camel case](https://pl.wikipedia.org/wiki/CamelCase) )